app/experimenter/nimbus-ui/src/components/PageEditBranches/index.tsx

```diff
@@ -36,6 +46,25 @@ const PageEditBranches: React.FunctionComponent<PageEditBranchesProps> = () => {
           }}
         />
         <h2 className="mt-3 h4">Branches</h2>
+        <p>
+          If you want, you can add a <strong>feature flag</strong> configuration
+          to each branch. Experiments can only change one flag at a time.{" "}
+          <a href={BRANCHES_DOC_URL}>Learn more</a>
+        </p>
+        {/* TODO: EXP-505 for accepting and saving edits to branches */}
+        <FormBranches
+          {...{
+            experiment,
+            // TODO: get this from initial GQL configuration fetch?
+            experimentFeatures,
+            // TODO: supply this as default value, track changes within FormBranches
+            equalRatio: false,
+            // TODO: figure out from where to get this value
+            featureSlug: "pinned_tabs",
+            // TODO: figure out how to determine this flag
+            featureHasValue: true,
```

Comment:
Scratching my head a bit at these pieces of data.

The branches include `featureValue` and `featureEnabled` but not the `slug` for which feature has been selected for manipulation in the experiment. 

And I see a list of features in `featureConfig` data from the new config GQL query. But, these items don't list whether any particular feature has a schema to allow a value.

Haven't dug into the server-side of things yet, wondering if we need to add some fields?

Reply:
Digging into the server-side and maybe answering some of my questions:

* The experiment model does already have `featureConfig` to be grabbed via GQL query - we're just not doing that yet, which is fixable.
* Something like this is probably what I want, in order to see schema for a feature. Seems like non-null schema is the answer for `featureHasValue`

```gql
{
  nimbusConfig {
    featureConfig {
      name
      slug
      id
      description
      schema
    }
  }
}
```

Reply:
@jodyheavener I'm still wrapping my head aroung GQL stuff, maybe you might have an insight here? If I add this to [GET_EXPERIMENT_QUERY](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/gql/experiments.ts#L23), I can fetch `featureConfig` for an experiment:

```gql
      featureConfig {
        slug
        name
        description
        application
        ownerEmail
        schema
      }
```

But, I don't see `featureConfig` in the generated type via `apollo codegen:generate`. 

As far as I can tell from the Django side and from playing in the GQL playground, it's in the schema as `featureConfig: NimbusFeatureConfigType`. 

But I'm not sure why it's not ending up in our [`getExperiment_experimentBySlug` type in `src/types/getExperiment.ts`](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/types/getExperiment.ts#L38-L55)

Reply:
Hmm, do you mean something like [this](https://gist.github.com/jodyheavener/09db471b0bd0bcda46cede5d9f430642)? You mentioned that the `types` command was giving you grief earlier, could that still be happening in this case?